src/courtroom.cpp

```diff
@@ -1372,6 +1370,14 @@ void Courtroom::done_received()
 
   show();
 
+  if (Options::getInstance().restoreWindowPositionEnabled()) {
+      auto maybe_point = Options::getInstance().windowPosition("courtroom");
+      if (maybe_point.has_value()) {
+          qDebug() << maybe_point.value();
+          QMainWindow::move(maybe_point.value());
```

Comment:
This doesn't take into account whatever the user may or may not have screens available for positioning.

In the case of some monitors, depending on the port used, the screen may or may not be listed as available when they are not switched on. This code has the side effect of potentially moving the windows out of reach without any way of figuring out how to restore it if they're not tech-savvy enough.

And no, we shouldn't rely on tech-support to let everybody know how certain key combinations or window manager may be used depending on the system they use.

Reply:
To provide some extra details, even if a monitor is plugged in but shut off, it will not always be listed as available. This depends on the manufacturer. Opening the client in this context could be placing the window... well literally anywhere on the desktop, not what was requested.